guides/doc-Updating_Project/master.adoc

```diff
@@ -28,3 +28,7 @@ endif::[]
 
 include::common/modules/proc_updating-smart-proxy.adoc[leveloffset=+1]
 endif::[]
+
+ifdef::satellite[]
```

Comment:
This should also work for Katello? (No idea about Foreman, especially on Debian).